### DISCUSSION TITLE
Schedule view should be extended

### DISCUSSION THREAD

**Discussion by Zentie13** [AzuraCast/AzuraCast#7520] [Ideas]
It is good to have the "schedule view" tab. However, it only shows one full week. It would be nice to have it customizable to see up to about two weeks in the future, or select a specific day.

**Comment by rogercreagh** [comment]
Or to show it as a simple list of forthcoming scheduled playlists with date and start and end times rather than in a calendar style frame which doesn't fit the screen. Next N scheduled playlists

**Reply by Zentie13** [reply]
You'll get no joy from these guys. They are simply set in their ways and always refer back to the end user as the problem. There are other platforms where the end user'comments are actually replied to and in many cases either solved or changed as per the suggestion. I suggest you move on.

**Reply by RM-FM** [reply]
> You'll get no joy from these guys. They are simply set in their ways and always refer back to the end user as the problem. There are other platforms where the end user'comments are actually replied to and in many cases either solved or changed as per the suggestion. I suggest you move on.

You know that Azuracast is a free, open-source project and essentially a one-man show, right? Buster is far from being closed off to new ideas. Please choose your words more carefully and respectfully. With such rude statements, you are genuinely risking the discontinuation of this project, as the main reason Buster continues to work on it is for the enjoyment and the satisfaction of providing a tool that countless radio creators deeply appreciate.

At this point, I’d like to sincerely thank Buster for his outstanding and relentless dedication to Azuracast.

**Reply by LordHelmchen666** [reply]
> You'll get no joy from these guys. They are simply set in their ways and always refer back to the end user as the problem. There are other platforms where the end user'comments are actually replied to and in many cases either solved or changed as per the suggestion. I suggest you move on.

Total disagree. Buster is doing great work. He is always open to add new features. Mostly in a short time. But he also needs some time for himself to regenerate.

**Reply by BusterNeece** [reply]
Zentie's behavior has gone from disrespectful and unfriendly to outright abusive. I won't tolerate it here.

I'm constantly making changes to this application. There's one or more commits every single day...for _eight years_ now. I've even wanted to spend some time on improving the schedule view (making it user-selectable like other reports), but I just haven't had time to. This isn't the way to get things done around here.

If you're finding this thread in the future and have recommendations for new features we can add, I'd suggest visiting https://features.azuracast.com/, our dedicated site for suggesting (and voting on) new feature requests.
